library/cohesity_agent.py

```diff
@@ -113,6 +127,25 @@
     cohesity_password: password
     download_location: /software/installers
     state: present
+
+# Install the current version of the agent on Linux using native installers, the service user here should be an
+# existing user
+- cohesity_agent:
+    server: cohesity.lab
+    cohesity_admin: admin
+    cohesity_password: password
+    state: present
+    service_user: cagent
+    native_package: True
+
+# Install the cohesity agent using native package downloaded from given uri, here cohesity cluster credentials are
```

Comment:
Install XXXX from given URI. Here, the Cohesity cluster credentials are not required.